apps/api-extractor/src/aedoc/ApiDocumentation.ts

```diff
@@ -377,8 +377,12 @@ export default class ApiDocumentation {
    * ensures that the reference is to an API item that is not 'Internal'.
    */
   private _completeLinks(): void {
-    while (this.incompleteLinks.length) {
-      const codeLink: IMarkupApiLink = this.incompleteLinks.pop();
+    for ( ; ; ) {
```

Comment:
>for ( ; ; ) [](start = 4, length = 11)

This is dangerous. Can you add an end condition?

Reply:
The end condition is on line 383.

I can change it to `while (true)` if you don't like the `for (;;)` notation.  I will need to disable a tslint rule.

---
In reply to: [151211677](https://github.com/Microsoft/web-build-tools/pull/442#discussion_r151211677) [](ancestors = 151211677)